layouts/learning/single.html

```diff
@@ -4,92 +4,102 @@
 
 {{ define "header" }}
 <div class="relative bg-[#1D65A6]">
-    {{ partial "site-nav.html" . }}
+  {{ partial "site-nav.html" . }}
```

Comment:
Code formatting changes. The only update made in this file was to add the `photo_credit` below.